.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,102 @@
+name: Tests / E2E
```

Comment:
everything in this file was just moved from `test.yml`. The only change is the addition of the `if` statements.